csrc/executor_utils.cpp

```diff
@@ -1281,8 +1312,7 @@ std::tuple<NvrtcFunction, std::string, std::vector<char>> nvrtcCompile(
     }
 
     const char* lowered_kernel_name = nullptr;
-    at::globalContext().getNVRTC().nvrtcGetLoweredName(
-        program, func_name.c_str(), &lowered_kernel_name);
+    nvrtcGetLoweredName(program, func_name.c_str(), &lowered_kernel_name);
```

Comment:
Don't we want to use `NVRTC_SAFE_CALL`?

Reply:
Good catch~~
I don't think we had nvrtc checks before, and I was only doing a string replacement. Let me revisit those